packages/rosaenlg-doc/doc/modules/ROOT/pages/changelog.adoc

```diff
@@ -19,6 +19,8 @@ https://keepachangelog.com/en/0.3.0/
 
 === Added
 
+* in `value` mixin, ability to generate a possesive determiner, using `personOwner: 1`, e.g. to generate _ma maison_, _ta maison_ in French. The `personOwner` parameter is also available in the `sentence` mixin subject and object groups.
```

Comment:
please indicate more clearly that `personOwner` only works in French (for both `value` and `sentence` mixins)

Reply:
ok